changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0 (Feature release)
+
+- Allow sharing of data between the server and client with `@financial-times/dotcom-ui-data-embed`.
+
+If you are using `@financial-times/dotcom-ui-app-context` to share data with the client you should now use the `@financial-times/dotcom-ui-data-embed` package instead.
+
+Please refer to the [`@financial-times/dotcom-ui-app-context` readme](https://github.com/Financial-Times/dotcom-page-kit/tree/master/packages/dotcom-ui-data-embed#financial-timesdotcom-ui-data-embed) for more details.
+
```

Comment:
It might also be worth adding the refactoring changes to the app context and flags packages? I don't think they're technically breaking changes as their interfaces haven't changed, but they're probably still significant

Reply:
I think we should only include things in the changelog which people will want to act on, so internal changes are not necessary, we could consider adding links to the pull requests for changes for people who are interested though.